client/src/ee/pages/embedded/integration/components/integration-header/IntegrationHeader.tsx

```diff
@@ -24,6 +23,7 @@ import {IntegrationTagKeys} from '@/ee/shared/queries/embedded/integrationTags.q
 import {IntegrationWorkflowKeys} from '@/ee/shared/queries/embedded/integrationWorkflows.queries';
 import {IntegrationKeys, useGetIntegrationQuery} from '@/ee/shared/queries/embedded/integrations.queries';
 import {WorkflowKeys, useGetWorkflowQuery} from '@/ee/shared/queries/embedded/workflows.queries';
+import DeleteWorkflowAlertDialog from '@/pages/automation/project/components/project-header/components/settings-menu/components/DeleteWorkflowAlertDialog';
```

Comment:
`embedded` shouldn't use anything from `automation`

I would keep the `IntegrationHeaderDeleteWorkflowAlertDialog` as changing it is not in the scope of this issue